client/deis.py

```diff
@@ -869,7 +869,8 @@ def auth_whoami(self, args):
         """
         user = self._settings.get('username')
         if user:
-            self._logger.info(user)
+            self._logger.info(
+                'You are {user} at {controller}'.format(user=user, controller=self._settings['controller']))
```

Comment:
Since we only support python 2.7, the positional arguments can be omitted. You can greatly simplify the logic here with:

```
'You are {} at {}'.format(user, self._settings['controller']))
```

Which also follows the style with the rest of the codebase.